LICENSE-3rdparty.csv

```diff
@@ -1,5 +1,6 @@
 Component,Origin,License,Copyright
 require,@types/node,MIT,Copyright Authors
+require,detect-libc,Apache license 2.0,Copyright Lovell Fuller
```

Comment:
Really not sure about this one. In the repo the license has a placeholder for the copyright so i just took his name on github. See: https://github.com/lovell/detect-libc/blob/master/LICENSE